ai-service/build.gradle

```diff
@@ -0,0 +1,22 @@
+dependencies {
+	compileOnly group: 'javax.websocket', name: 'javax.websocket-api', version: '1.1'
+}
+
+ext {
+   httpPort = 8083
+   httpsPort = 8275
+   appUrl = "http://${hostname}:${httpPort}/ai"
+}
+
+liberty {
+    server {
+        name = "ai-service"
+        dropins = [war]
+        configDirectory = file('src/main/liberty/config')
+        jvmOptions = ['-Dorg.libertybikes.ai.restclient.LibertyBikesClient/mp-rest/url=http://localhost:8080']
```

Comment:
add a TODO here that we will need to make this configurable, because the game-service may be on a different machine than the AI service